cartridge/webui/api-config.lua

```diff
@@ -252,6 +266,66 @@ local function set_sections(_, args)
     return get_sections(nil, {sections = query_sections})
 end
 
+local function validate_config(_, args)
+    if confapplier.get_readonly() == nil then
+        return nil, ValidateConfigError:new(
+            "Current instance isn't bootstrapped yet"
+        )
+    end
+
+    if args.sections == nil then
+        args.sections = {}
+    end
+
+    local patch = {}
+    for _, input in ipairs(args.sections) do
+        patch[input.filename] = input.content or box.NULL
+    end
+
+    -- Check schema if present
+    if patch['schema.yml'] then
+        local ddl_manager = assert(service_registry.get('ddl-manager'))
+        local ok, err = ddl_manager.check_schema_yaml(args.as_yaml)
+        if not ok then
+            if err.class_name == ddl_manager.CheckSchemaError.name then
+                return { error = err.err }
+            else
+                return nil, err
+            end
+        end
+    end
```

Comment:
Please, leave a comment on why it's necessary. The schema is already checked as a part of `confapplier.validate_config`, but it's skipped on non-leader instances. That's why we use `ddl_manager.check_schema_yaml` (which uses rpc)

Reply:
It also could be optimized with an `if not is_leader ...`

Reply:
Done!